pkg/ocm/helpers.go

```diff
@@ -354,9 +356,29 @@ func (c *Client) LinkOrgToRole(orgID string, roleARN string) (bool, error) {
 	return true, nil
 }
 
+func (c *Client) GetLinkedRoles(orgID string) ([]string, error) {
+	resp, err := c.ocm.AccountsMgmt().V1().Organizations().Organization(orgID).
+		Labels().Labels(STSOCMRole).Get().Send()
+	if err != nil && resp.Status() != http.StatusNotFound {
+		return nil, err
+	}
+
+	return strings.Split(resp.Body().Value(), ","), nil
+}
+
+func (c *Client) IsRoleLinked(roleARN string, linkedRoles []string) bool {
```

Comment:
I don't think this should belong in ocm client. This doesn't go to OCM.

I also think this can be very generic func implementation `contains`, that might be used in other places. Maybe you can place it in a helper file.
See https://freshman.tech/snippets/go/check-if-slice-contains-element/

Reply:
Good catch!